tensorboard/plugins/histogram/histograms_demo.py

```diff
@@ -38,35 +39,54 @@ def run_all(logdir, verbose=False):
   # Make a normal distribution, with a shifting mean
   mean_moving_normal = tf.random_normal(shape=[1000], mean=(5*k), stddev=1)
   # Record that distribution into a histogram summary
-  histogram_summary.op("normal/moving_mean", mean_moving_normal)
+  histogram_summary.op("normal/moving_mean",
+                       mean_moving_normal,
+                       description="A normal distribution whose mean changes "
+                                   "over time.")
 
   # Make a normal distribution with shrinking variance
   shrinking_normal = tf.random_normal(shape=[1000], mean=0, stddev=1-(k))
   # Record that distribution too
-  histogram_summary.op("normal/shrinking_variance", shrinking_normal)
+  histogram_summary.op("normal/shrinking_variance", shrinking_normal,
+                       description="A normal distribution whose variance "
+                                   "shrinks over time.")
 
   # Let's combine both of those distributions into one dataset
   normal_combined = tf.concat([mean_moving_normal, shrinking_normal], 0)
   # We add another histogram summary to record the combined distribution
-  histogram_summary.op("normal/bimodal", normal_combined)
+  histogram_summary.op("normal/bimodal", normal_combined,
+                       description="A combination of two normal distributions, "
+                                   "one with a moving mean and one with  "
+                                   "shrinking variance. The result is a "
+                                   "distribution that starts as unimodal and "
+                                   "becomes more and more bimodal over time.")
 
   # Add a gamma distribution
   gamma = tf.random_gamma(shape=[1000], alpha=k)
-  histogram_summary.op("gamma", gamma)
+  histogram_summary.op("gamma", gamma,
+                       description="A gamma distribution whose shape "
```

Comment:
Would it be possible to put these HTML and documentation changes in separate PRs?

I'll also note that using non-ascii characters in Python 6 sources is really hard to do correctly without introducing risk. This would be a UTF-8 bytestring on Python 2, which might not work on non-utf8 terminals. I would recommend latin a or `u'... \u03b1 ...'`.